src/service/receipt.service.ts

```diff
@@ -41,41 +60,53 @@ export class ReceiptService {
     public formatTransactionStatements(statement: any): string {
         let txt = '';
         txt += chalk.green('transactionStatements:\t') + '\n';
-        txt += '-'.repeat('transactionStatements:\t'.length) + '\n\n';
         statement.transactionStatements.map((transaction: TransactionStatement, transactionIndex: number) => {
-            txt += 'height:\t\t' + transaction.height + '\n';
             transaction.receipts.map((receipt: any, receiptIndex: number) => {
-                txt += '<index: ' + transactionIndex + '-' + receiptIndex + '>\t';
                 if (receipt instanceof BalanceTransferReceipt) {
-                    txt += 'version:\t' + receipt.version + '\n';
-                    txt += '\t\ttype:\t\t' + ReceiptType[receipt.type]  + '\n';
-                    txt += '\t\trecipientAddress:\t' +
+                    this.transactionStatementsTable.push([
+                        receipt.version as Table.CellOptions,
```

Comment:
What does as Table.CellOptions mean?

Reply:
The instance of `receipt .version` is `ReceiptType` which can not cast to CellOptions. So I cast it to Table.CellOptions Mandatorily. 
In fact, the value of `receipt.version` is 1 which is different from `ReceiptType`.

Reply:
I believe you can cast version (integer) as a string (receipt.version.toString())